CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # splat Release Notes
 
+### 0.33.3
```

Comment:
so are we doing 0.33.3 or 0.34.0? The other comment was resolved without any comments